File: Day036_2025/Q036_GFG.java
class Q036_GFG {
    // Function to convert a binary tree into its mirror tree.
    void mirror(Node node) {
        if (node == null) {
            return;
        }

        // Swap left and right children
        Node temp = node.left;
        node.left = node.right;
        node.right = temp;

        // Recursively call for left and right subtrees
        mirror(node.left);
        mirror(node.right);
    }
}

File: Day036_2025/Q036_GFG.py
class Solution:
    # Function to convert a binary tree into its mirror tree.
    def mirror(self, root):
        if root is None:
            return
        
        # Swap left and right children
        root.left, root.right = root.right, root.left

        # Recursively call for left and right subtrees
        self.mirror(root.left)
        self.mirror(root.right)
